dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/post-fee-record-correction.controller/index.ts

```diff
@@ -64,6 +65,10 @@ export const postFeeRecordCorrection = async (req: PostFeeRecordCorrectionReques
         throw new NotFoundError(`Failed to find a fee record with id ${feeRecordId} and report id ${reportId}`);
       }
 
+      const { teamName, emails } = await getBankPaymentOfficerTeamDetails(feeRecord.report.bankId);
```

Comment:
We want to be certain that the emails we are saving are the same ones we are sending the emails too, so rather than fetch them here and then separately fetch them again inside `sendFeeRecordCorrectionRequestEmails` on line 96, we should refactor `sendFeeRecordCorrectionRequestEmails` to have params for these values and not look them up itself and then pass these into it when we call it on line 96.

Reply:
refactored so teamName and emails are passed from here and not refetched